### PR TITLE
Prisma Cloud Compute Reporting Pack

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This pack creates PCC reports for compliance and vulnerabilities which is currently a gap in the Prisma product.  Today many customers are using https://github.com/timekillerj/pcs-compute-report-generator 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrismaCloudComputeReporting/Scripts/PrismaCloudComputeCreateImpactTableForCVE/PrismaCloudComputeCreateImpactTableForCVE.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+# Create CVE Host table from running: !prisma-cloud-compute-vulnerabilities-impacted-resources-list and setting to indicator (custom script)
+
```
Comment (bziser) [inline Packs/PrismaCloudComputeReporting/Scripts/PrismaCloudComputeCreateImpactTableForCVE/PrismaCloudComputeCreateImpactTableForCVE.py:4]: Please use main() function

File: Packs/PrismaCloudComputeReporting/Scripts/PrismaCloudComputeCreateImpactTableForCVE/PrismaCloudComputeCreateImpactTableForCVE.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+if "indicator" in demisto.args():
```
Comment (bziser) [inline Packs/PrismaCloudComputeReporting/Scripts/PrismaCloudComputeCreateImpactTableForCVE/PrismaCloudComputeCreateImpactTableForCVE.py:5]: Avoid initializing parameters/script args in the global space. Initialize them in main.

**Review by bziser — COMMENTED** [review]
From the scripts side, it looks good. 
@tomer-pan Please review the playbooks.
@nericksen Please add the TestPlaybooks in Tests/conf.json, add unit-test to the scripts and let's set up a demo with @tomer-pan

File: Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        simple: ${CreatedIndicator.Value}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Create a markdown table of impacted images and hosts based on Prisma Cloud Compute CVE data.
+      id: 49ca785f-8c72-4efd-8e6a-f774cade1445
+      iscommand: false
+      name: Create Impact Table on CVE based on Prisma Cloud Compute Data
+      script: a0e80035-f917-4ad5-8236-54904ad5d6e6
```
Comment (tomer-pan) [inline Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml:135]: @nericksen  Please add the correct script values under the "name" and "script" keys.

File: Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+        simple: ${CreatedIndicator.Value}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Create a Vulnerability summary Report from XSOAR CVEs.  These CVEs can be created from the command !prisma-cloud-compute-images-scan-list
+      id: ca631fd9-109b-4e32-8302-1ce76e711330
+      iscommand: false
+      name: Generate Vulnerability Report
+      script: 30ce4c34-e8ae-49b6-8a5b-2c23f30305bf
```
Comment (tomer-pan) [inline Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml:169]: @nericksen Please add the correct script name.

File: Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+        simple: ${PrismaCloudCompute.ReportsImagesScan}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Create a Compliance summary Report from Prisma Cloud Compute image data.
+      id: 372bf96c-813e-4577-80e9-584ef14a47aa
+      iscommand: false
+      name: Generate Compliance Report
+      script: 9b9dbe0c-1a86-4f05-82d7-fe404ad8e8ca
```
Comment (tomer-pan) [inline Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting.yml:202]: @nericksen Please add the correct script name.

File: Packs/PrismaCloudComputeReporting/IncidentTypes/incidenttype-Prisma_Cloud_Compute_Reporting.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    "detached": false,
+    "disabled": false,
+    "extractSettings": {
+        "fieldCliNameToExtractSettings": {},
+        "mode": "Specific"
+    },
+    "hours": 0,
+    "hoursR": 0,
+    "id": "Prisma Cloud Compute Reporting",
+    "layout": "c59238c1-9445-4dc0-8af2-33f8603f2a43",
```
Comment (tomer-pan) [inline Packs/PrismaCloudComputeReporting/IncidentTypes/incidenttype-Prisma_Cloud_Compute_Reporting.json:16]: @nericksen Please add the correct layout name.

File: Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting_README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
+This playbook creates a summary report of the vulnerabilities and compliance issues found within a Prisma Cloud Compute deployment. The reports are generated in html format and can therefore be sent via email and other methods.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+This playbook does not use any sub-playbooks.
+
+### Integrations
+* PaloAltoNetworks_PrismaCloudCompute
```
Comment (tomer-pan) [inline Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting_README.md:10]: @nericksen I can't find the integration files in this PR

Comment (nericksen) [reply Packs/PrismaCloudComputeReporting/Playbooks/Prisma_Cloud_Compute_Vulnerability_and_Compliance_Reporting_README.md:10]: removed

**Comment by bziser** [comment]
@nericksen Thanks for the great demo. Before I approve the PR, I will wait for the unit test.

**Pull request merged**